packages/db/prisma/schema.prisma

```diff
@@ -4,7 +4,7 @@
 generator client {
   provider        = "prisma-client-js"
   previewFeatures = ["tracing", "metrics"]
-  binaryTargets = ["native", "debian-openssl-3.0.x"]
+  binaryTargets   = ["native", "linux-arm64-openssl-3.0.x"]
```

Comment:
please leave debian-openssl-3.0.x

Reply:
Actually. It looks like the rest api build is not complaining anymore about these missing binaries. Maybe we can remove this line now as it adds an extra unused binary to the image 🤔 ?